packages/devtools/lib/src/service_manager.dart

```diff
@@ -181,10 +186,11 @@ class ServiceConnectionManager {
       try {
         await service.streamListen(id);
       } catch (e) {
-        // Don't complain about '_Logging' or 'Logging' events (newer VMs don't
-        // support '_Logging' and older VMs don't support 'Logging').
         // TODO(devoncarew): Remove this check on or after approx. Oct 1 2019.
-        if (!id.endsWith('Logging')) {
+        if (id.endsWith('Logging') || id.endsWith('Service')) {
+          // Don't complain about '_Logging' or 'Logging' events (newer VMs don't
```

Comment:
nit: update comment to include both _Logging and _Service

Reply:
updated